### PR TITLE
Add timeGutterHeaderComponent

### LINKED ISSUE #853: Specify rbc-time-header-gutter content
#### Do you want to request a _feature_ or report a _bug_?

_Feature_

#### What's the current behavior?

`rbc-time-header-gutter` is always empty.

#### What's the expected behavior?

Add new prop to specify the text (at least) content of `rbc-time-header-gutter`

### PR DESCRIPTION
**Pull request by danbovey**
Hi

This change adds a new component option that gets inserted as the contents of the header of the time gutter in week and day views.

Closes #853

Added an example of using the component to "Custom View" in examples:

image

### CODE REVIEW

File: examples/demos/customView.js [hunk 1]
```diff
@@ -52,6 +52,9 @@ let CustomView = () => (
     defaultView="week"
     defaultDate={new Date(2015, 3, 1)}
     views={{ month: true, week: MyWeek }}
+    components={{
+      timeGutterHeader: <p>Custom gutter text</p>,
+    }}
```
Comment (jquense) [inline examples/demos/customView.js]: this is a good place for this. please add a storybook story instead for testing

**Comment by danbovey** [comment]
Before you merge this, I think this new component option acts differently to all the other options. It doesn't allow the component to be passed in as a variable. i.e.

```js
components={{
  timeGutterHeader: TimeGutterHeader
}}
```

Working on an update.

File: stories/Calendar.js [hunk 2]
```diff
@@ -561,3 +561,22 @@ storiesOf('module.Calendar.week', module)
...
+      <div style={{ height: 600 }}>
+        <Calendar
+          popup
+          events={demoEvents}
+          onSelectEvent={action('event selected')}
+          defaultDate={new Date(2015, 3, 1)}
+          defaultView="week"
+          views={['week', 'day']}
+          components={{
+            timeGutterHeader: TimeGutter,
```
Comment (jquense) [inline stories/Calendar.js]: please also update the propTypes in Calendar with appropriate documentation as well

Comment (danbovey) [reply stories/Calendar.js]: Yeah sure 😃

**Comment by danbovey** [comment]
GitHub is only blocking the merge because there aren't any changes to the lines you made the last comment on. All the requested changes are done.

**Comment by jquense** [comment]
thanks!

**Comment by akazakou** [comment]
@jquense Could you please create a new version of npm package, that included this feature?

**Pull request merged**